### DISCUSSION TITLE
Offer a field for a Mastodon profile link in the user profile

### DISCUSSION THREAD

**Discussion by ckane** [community/community#38374] [Other Feature Feedback, Questions, & Ideas]
There's a Twitter profile space in there, but I think GitHub should now add a link for Mastodon accounts, as well.

**Comment by jfullstackdev** [comment]
as for unlimited links, you can use the default README on your landing page to include all of your links, you can even make it beautiful by making a link icon instead of the actual link

this is what it looks like 

![image](https://user-images.githubusercontent.com/47092464/200153781-ed3ef9f3-3fd5-4e86-809b-32ca850920af.png)

and there are so many external services you can use for this default README of GitHub, like the GitHub Stats that I'm using,

just make a repo of the same name as your username and you will be prompted with that README,

**Comment by jose19961975** [comment]
Ok

**Comment by jankatins** [comment]
This won't work completely, as the `rel="me"` art needed to verify the link from mastodon, is repalced by a `rel="nofollow"`:-(

**Reply by hxm-cpp** [reply]
only way to make it work is by linking it your github profile.

**Comment by derekprior** [comment]
Hey folks, I just wanted to give you a heads up that we are working on this. It'll ship when it's ready, but with holiday season vacations coming up that won't be for a bit yet. Thanks for your patience.
